corehq/apps/reports/standard/__init__.py

```diff
@@ -314,6 +315,7 @@ def datespan(self):
     @property
     def default_datespan(self):
         datespan = DateSpan.since(self.datespan_default_days, timezone=self.timezone, inclusive=self.inclusive)
+        datespan.max_days = self.datespan_max_days
```

Comment:
@czue, I set `max_days` as an attribute here because we're using `DateSpan.since()` to get `datespan` instead of using the constructor. It didn't feel right passing `max_days` as a parameter of `since()`.